services/bundle_analysis.py

```diff
@@ -66,6 +72,61 @@ def get_extension(filename: str) -> str:
     return file_extension
 
 
+class BundleAnalysisMeasurementsAssetType(enum.Enum):
+    REPORT_SIZE = MeasurementName.BUNDLE_ANALYSIS_REPORT_SIZE
+    JAVASCRIPT_SIZE = MeasurementName.BUNDLE_ANALYSIS_JAVASCRIPT_SIZE
+    STYLESHEET_SIZE = MeasurementName.BUNDLE_ANALYSIS_STYLESHEET_SIZE
+    FONT_SIZE = MeasurementName.BUNDLE_ANALYSIS_FONT_SIZE
+    IMAGE_SIZE = MeasurementName.BUNDLE_ANALYSIS_IMAGE_SIZE
+    ASSET_SIZE = MeasurementName.BUNDLE_ANALYSIS_ASSET_SIZE
+
+
+class BundleAnalysisMeasurementData(object):
+    def __init__(
+        self,
+        raw_measurements: List[dict],
+        asset_type: BundleAnalysisMeasurementsAssetType,
+        asset_name: Optional[str],
+        interval: Interval,
+        after: datetime,
+        before: datetime,
+    ):
+        self.raw_measurements = raw_measurements
+        self.measurement_type = asset_type
+        self.measurement_name = asset_name
+        self.interval = interval
+        self.after = after
+        self.before = before
+
+    @cached_property
+    def asset_type(self) -> str:
+        return self.measurement_type.name
```

Comment:
Is there value in these properties being in Shared over api? Are these used in worker/the PR comment?

Reply:
Shouldn't be needed for worker, unless we can support trend lines in PR comments some day 😮